transport_nantes/photo/templates/photo/single_entry.html

```diff
@@ -0,0 +1,144 @@
+{% extends 'asso_tn/base_mobilitain.html' %}
+{% load markdown %}
+{% load static %}
+{% load compress %}
+{% load crispy_forms_tags %}
+
+{% block localscripts %}
+    {% compress js %}
+        <script src="{% static 'photo/js/vote_system.js' %}" defer></script>
+        <script type="text/javascript">
+            {% comment %}Those values are used in vote_system.js{% endcomment %}
+            {% if has_voted %}
```

Comment:
This presents four different js files to the caching system, which increases the chances of cache miss.

Maybe we need to do that, but why aren't these values simply arguments to the function?

Reply:
I count 1 file and 1 inline script. 

I don't understand what you mean, these variables are state. They are used in various vote_system.js functions to make it run properly. 

Could you please expand on the cache miss issue ? Do you suggest we don't compress the JS ?

Reply:
Do you still not understand this?